omod/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -122,7 +116,6 @@ public void shouldCreateAVisitWithEncounters() throws Exception {
 		int originalCount = service.getAllVisits().size();
 		final String patientUuid = "5946f880-b197-400b-9caa-a3c661d23041";
 		Patient patient = Context.getPatientService().getPatientByUuid(patientUuid);
-		Assert.assertEquals(0, service.getVisitsByPatient(patient).size());
```

Comment:
Instead of removing these assertions, shouldn't you instead change them to the expected count?